src/yadism/runner.py

```diff
@@ -95,7 +95,10 @@ def __init__(self, theory: dict, observables: dict):
         self._observables = new_observables
 
         # Setup eko stuffs
-        interpolator = InterpolatorDispatcher.from_dict(self._observables, mode_N=False)
+        xgrid = XGrid(self._observables["interpolation_xgrid"])
```

Comment:
Actually, even here: you should read the "logginess" from the run card ...

Reply:
Standard key has been `"interpolation_is_log"` until now. We need a rework of the runcard also for yadism, but not now

Reply:
I should have solved it!